core/trino-main/src/main/java/io/trino/server/protocol/Query.java

```diff
@@ -751,35 +754,23 @@ private static StatementStats toStatementStats(QueryInfo queryInfo)
                 .setPhysicalInputBytes(queryStats.getPhysicalInputDataSize().toBytes())
                 .setPeakMemoryBytes(queryStats.getPeakUserMemoryReservation().toBytes())
                 .setSpilledBytes(queryStats.getSpilledDataSize().toBytes())
-                .setRootStage(toStageStats(outputStage))
+                .setRootStage(rootStageStats)
                 .build();
     }
 
-    private static StageStats toStageStats(StageInfo stageInfo)
+    private static StageStats toStageStats(StageInfo stageInfo, Set<String> globalUniqueNodes)
```

Comment:
Do existing tests cover this change ? If not, please add them

Reply:
Test added